### PR TITLE
Feat/case study/get

### PR DESCRIPTION
**Pull request by chiranshu18**

### CODE REVIEW

**Comment by rohini-senthil** [comment]
There's a lot of linting changes – can we revert all those file?

File: src/controllers/case-study.controller.js [hunk 1]
```diff
@@ -35,7 +35,33 @@ const deleteCaseStudy = async (req, res) => {
   }
 };
 
+const getCaseStudy = async (req, res) => {
+  try {
+    const { id } = req.params;
+    const caseStudy = await caseStudyServices.getCaseStudy(id);
```
Comment (rohini-senthil) [inline src/controllers/case-study.controller.js:41]: Can we add logs using our logger module like how it's done for other routes

File: tests/controllers/case-study.controller.test.js [hunk 2]
```diff
@@ -44,4 +44,34 @@ describe('CaseStudyController', () => {
       expect(mockData.toDelete.mockRes.json).toHaveBeenCalledWith({ message: 'Something went wrong' });
     });
   });
+
+  describe('getCaseStudyController', () => {
```
Comment (rohini-senthil) [inline tests/controllers/case-study.controller.test.js:48]: Let's also include service tests for the two methods

File: src/controllers/case-study.controller.js (hunk 1 above)
Comment (chiranshu18) [reply src/controllers/case-study.controller.js:41]: okay.

File: tests/controllers/case-study.controller.test.js (hunk 2 above)
Comment (chiranshu18) [reply tests/controllers/case-study.controller.test.js:48]: sure.

File: database/config/config.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "development": {
     "username": "postgres",
     "password": null,
-    "database": "staffing_pipeline_db",
+    "database": "staffing_pipeline_DB",
```
Comment (Chetan-MckinseyBEBR) [inline database/config/config.json:5]: Is this change in DB name agreed with the team?

Comment (rohini-senthil) [reply database/config/config.json:5]: Nope, we're going with `staffing_pipeline_db`
Also @chiranshu18, let's remove all linting changes on this PR

**Review by rohini-senthil — APPROVED** [review]

**Pull request merged**